docs/source/nlp/megatron.rst

```diff
@@ -1,183 +0,0 @@
-.. _megatron_finetuning:
-
-NeMo Megatron
-=============
```

Comment:
why this doc was deleted?

Reply:
becaused observed a warning:
```
checking consistency... /NeMo_fork/docs/source/nlp/megatron.rst: WARNING: document isn't included in any toctree done
```

When diving deep into `nlp/megatron.rst`, it is old because we support 4 types of parallelisms now. I discussed this with @ericharper yesterday. NLP team recently updated the megatron docs and `nlp/nemo_megatron` folder includes all up-to-date docs. 

@okuchaiev could you pls also confirm this again?

Reply:
let me revert the removal of `megatron.rst` to unblock this PR cause @ericharper is working on the fix for megatron doc.